comCam/manual_defects/README.md

```diff
@@ -0,0 +1,9 @@
+This directory contains manual defects for the LSSTComCam instrument.
+The LSSTComCam focal plane consists of one raft of ITL sensors.
+As we do not have QE scans of LSSTComCam, we have simply taken an average of all the ITL sensors from the LSSTCam sensor QE scans as processed in DM-40164.
+
+Data Processing
+---------------
+
+Look at ingest_manual_defects.py attached to https://rubinobs.atlassian.net/browse/DM-47365
```

Comment:
Please copy this script to `obs_lsst` to match up with how we've handled the transmission curve.